app/views/candidate_interface/degrees/base/new.html.erb

```diff
@@ -25,6 +25,20 @@
         <%= f.govuk_text_field :qualification_type, label: { text: t('application_form.degree.qualification_type.label'), size: 'm' }, hint_text: t('application_form.degree.qualification_type.hint_text.new') %>
         <%= f.govuk_text_field :subject, label: { text: t('application_form.degree.subject.label'), size: 'm' } %>
         <%= f.govuk_text_field :institution_name, label: { text: t('application_form.degree.institution_name.label'), size: 'm' } %>
+        <%= f.govuk_radio_buttons_fieldset :grade, legend: { text: t('application_form.degree.grade.label'), tag: 'span' } do %>
+          <%= f.govuk_radio_button :grade, 'first', label: { text: t('application_form.degree.grade.first.label') } %>
+          <%= f.govuk_radio_button :grade, 'upper_second', label: { text: t('application_form.degree.grade.upper_second.label') } %>
+          <%= f.govuk_radio_button :grade, 'lower_second', label: { text: t('application_form.degree.grade.lower_second.label') } %>
+          <%= f.govuk_radio_button :grade, 'third', label: { text: t('application_form.degree.grade.third.label') } %>
+          <%= f.govuk_radio_button :grade, 'other', label: { text: t('application_form.degree.grade.other.label') } do %>
+            <%= f.govuk_text_field :other_grade, label: { text: t('application_form.degree.grade.other.conditional.label') }, width: 10 %>
+          <% end %>
+          <%= f.govuk_radio_divider %>
+          <%= f.govuk_radio_button :grade, 'predicted', label: { text: t('application_form.degree.grade.predicted.label') } do %>
+            <%= f.govuk_text_field :predicted_grade, label: { text: t('application_form.degree.grade.predicted.conditional.label') }, hint_text: t('application_form.degree.grade.predicted.conditional.hint_text'), width: 10 %>
+          <% end %>
+        <% end %>
+        <%= f.govuk_text_field :award_year, label: { text: t('application_form.degree.award_year.label'), size: 'm' }, hint_text: t('application_form.degree.award_year.hint_text'), width: 4 %>
```

Comment:
Stuff like this reaaaally makes me appreciate the awesome work that @peteryates put into the form builder; without it, this would be a lot more code!

Reply:
Definitely! So much easier to create. 🚀